rtdata/languages/Deutsch

```diff
@@ -2222,13 +2223,13 @@ ZOOMPANEL_ZOOMOUT;Herauszoomen\nTaste: <b>-</b>
 ! Untranslated keys follow; remove the ! prefix after an entry is translated.
 !!!!!!!!!!!!!!!!!!!!!!!!!
 
-!EXIFPANEL_SHOWALL;Show all
-!QINFO_FRAMECOUNT;%2 frames
-!QINFO_HDR;HDR / %2 frame(s)
-!QINFO_PIXELSHIFT;PixelShift / %2 frame(s)
-!SAMPLEFORMAT_0;Unknown data format
-!SAMPLEFORMAT_1;Unsigned 8 bits
-!SAMPLEFORMAT_2;Unsigned 16 bits
-!SAMPLEFORMAT_4;LogLuv 24 bits
-!SAMPLEFORMAT_8;LogLuv 32 bits
-!SAMPLEFORMAT_16;32 bits floating point
+EXIFPANEL_SHOWALL;Alle anzeigen
+QINFO_FRAMECOUNT;%2 Frames
+QINFO_HDR;HDR / %2 Frame(s)
+QINFO_PIXELSHIFT;Pixel Shift / %2 Frame(s)
+SAMPLEFORMAT_0;Umbekanntes Format
```

Comment:
@TooWaBoo There's a typo in `Umbekanntes`

Reply:
Argh, thanks, I will close this PR, 'cause I've found bug in exifpanel.cc which affects the RT and TooWaBoo theme and create a new pull request.